gui/control_main.py

```diff
@@ -3834,11 +3907,19 @@ def pixelSelect(self, event):
                 self.drawInteractiveRasterCB()
             return
         fov = self.getCurrentFOV()
-        correctedC2C_x = self.getBeamCenterX() + (
-            x_click - (self.centerMarker.x() - self.centerMarkerCharOffsetX) - 20
-        )
-        correctedC2C_y = self.getBeamCenterY() + (
-            y_click - (self.centerMarker.y() - self.centerMarkerCharOffsetY) - 40
+        if daq_utils.beamline == "nyx":
```

Comment:
@mskinner5278 This is where the carve out for nyx is for the click to center code. Please make changes to whatever you have at NYX